cmd/kubeapps-apis/plugins/resources/v1alpha1/server.go

```diff
@@ -416,13 +416,16 @@ func resourceRefsEqual(r1, r2 *pkgsGRPCv1alpha1.ResourceRef) bool {
 
 // errorByStatus generates a meaningful error message
 func errorByStatus(verb, resource, identifier string, err error) error {
+	// If the error is already a k8s StatusError, return the StatusErr
```

Comment:
@absoludity Is this a TO-DO comment?

Reply:
Yes - it was a note to myself but something I didn't end up doing. Removed, thanks!